assertk-jvm/src/test/kotlin/test/assertk/assertions/support/NumberSpec.kt

```diff
@@ -0,0 +1,89 @@
+package test.assertk.assertions.support
+
+import assertk.assertions.isNegative
+import assertk.assertions.isPositive
+import assertk.assertions.isZero
+import org.junit.Test
+import java.math.BigDecimal
+import java.math.BigInteger
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+
+
+class BigDecimalSpecNumberOnIsZero {
```

Comment:
Can you update the tests? should be able to just add them in `NumberTest`

Reply:
They can't be moved to `NumberTest`, that test is environment agnostic and doesn't support `BigDecimal` or `BigInteger`

Reply:
In that case, make `JavaNumberTest`.

Reply:
Done